CHANGELOG.md

```diff
@@ -19,6 +19,14 @@ All Notable changes to `Backpack CRUD` will be documented in this file
 ### Security
 - Nothing
 
+## [3.2.14] - 2017-07-xx
+
+## Added
+- Brasilian Portugese translation, thanks to [Guilherme Augusto Henschel](https://github.com/cenoura);
+
+## Fixed
+- setFromDb() now uses the column name as index, for afterColumn() and beforeColumn() to work with it - #759;
```

Comment:
The English here is a little non-idiomatic (and I'm not really sure straight away what it means).